### DISCUSSION TITLE
SQLite Cloud…?

### DISCUSSION THREAD

**Discussion by slanelb** [pocketbase/pocketbase#5043] [Q&A] [answered]
Hello,
I wondered if the dream of a distributed version of pocketbase may be possible soon with the launch of [SQLite Cloud](https://sqlitecloud.io/)?
Would love to hear thoughts on this.
Thank you for a wonderful project (Pocketbase).

**Comment by ganigeorgiev** [comment, marked as answer]
See https://github.com/pocketbase/pocketbase/issues/5038.

**Reply by slanelb** [reply]
Sorry, I did a quick search and couldn’t find it. Sorry about that.
